test/C/src/federated/transient/TransientHotSwap.lf

```diff
@@ -0,0 +1,153 @@
+/**
+ * This LF program is a variant of TransientDownstreamWithTimer that tests the Hot Swap mechanism.
+ * For this, it tests if the transient's downstream executes as expected. In this test:
+ * - the transient federate DO NOT spontaneously leave the federation.
+ * - the downstream of the transient federate has only one transient as upstream.
+ * - A persistent federate `HotSwapMiddle` executes `mid` after 1s to launch the hot swap.
+ */
+target C {
+  timeout: 2 s
+}
+
+preamble {=
+  #include <stdlib.h>
+  #include <string.h>
+=}
+
+/** Persistent federate that is responsible for lauching the transient federate */
+reactor TransientExec(offset: time = 0, period: time = 0, fed_instance_name: char* = "instance") {
```

Comment:
Same comment about `offset` and `period`.  These seem identical across files. Wouldn't it be better to import them from just one file?

Reply:
When `TransientExec` is imported, the execution fails with a segmentation fault when launching `mid`.
Having a `char *` as a parameter (`fed_instance_name`) when instantiating an imported federate causes this. 
Any idea about how to fix this?